src/js/components/Form/Form.js

```diff
@@ -334,6 +353,23 @@ const Form = forwardRef(
       }
     }, [applyValidationRules, touched]);
 
+    useEffect(() => {
+      const element = formRef.current;
+      analyticsRef.current = { start: new Date(), errors: {} };
+      sendAnalytics({
+        type: 'formOpen',
+        element,
+      });
+      return () => {
+        sendAnalytics({
```

Comment:
I'm wondering if we can optimize to only send this when the form wasn't submitted? It seems a bit redundant to send both "formSubmit" and "formClose" with similar content.

Reply:
Great idea.

Reply:
Agree. I was wanting to be able to differentiate between whether the form closed because of a successful event vs. an abandonment due to errors vs. just didn't need it (e.g. it mounted but I navigated away).